kairon/api/app/routers/pos.py

```
@@ -0,0 +1,128 @@
+        client_name=req.client_name,
+        bot=current_user.get_bot(),
+        user=current_user.get_user()
+    )
+
+    return Response(data=data)
+
+
+@router.delete("/client/delete", response_model=Response)
+def delete_client(
+        req: ClientRequest,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=ADMIN_ACCESS)
```

Ruff (0.14.5)

Unused function argument: `current_user`

(ARG001)